Feature/MTM-38666 update documentation about restricted feature prefix for app name (#435)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/managing-applications.md

<br><br>
<img src="/images/users-guide/Administration/admin-application-duplicate-2.png" alt="Duplicate application">
<br><br>

> > **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name.

Reviewer: done

> > **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix for the application name.

Reviewer 2: ```suggestion
> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name.
```

> > **Info:** Prefix 'feature-' is restricted by the platform from being used. You will not be able to create applications using this prefix for application name.

Reviewer 3: ```suggestion
> **Info:** The prefix 'feature-' is restricted by the platform from being used. You will not be able to create applications using this prefix for the application name.
```

<br><br>
<img src="/images/users-guide/Administration/admin-application-duplicate-2.png" alt="Duplicate application">
<br><br>

> > **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name. This also applies to existing applications for use cases when duplicate application feature is used.

Reviewer 2: ```suggestion
> **Info:** The platform restricts the use of the prefix "feature-". You cannot create applications using this prefix in the application name. This also applies to existing applications in cases where the duplicate application feature is used.
```
